bindings/python/src/pipeline/node/IMUBindings.cpp

```
@@ -29,6 +29,7 @@ void bind_imu(pybind11::module& m, void* pCallstack) {
 
     // Properties
     imuSensor.value("ACCELEROMETER_RAW", IMUSensor::ACCELEROMETER_RAW, DOC(dai, IMUSensor, ACCELEROMETER_RAW))
+        .value("ACCELEROMETER_UNCALIBRATED", IMUSensor::ACCELEROMETER_UNCALIBRATED, "Accelerometer in the unified DepthAI IMU frame without affine calibration.")
```

Cppcheck (2.20.0)

[information] Include file

(missingIncludeSystem)